tests/test_api.py

```diff
@@ -199,6 +212,18 @@ def test_gml_extract_bbox_time():
     assert result['tbox'] == ['2005-12-31', '2013-11-30']
 
 
+def test_jpge_2000_extract_bbox():
+    result = geoextent.fromFile("tests/testdata/jpge2000/MSK_SNWPRB_60m.jp2", bbox=True)
+    assert result['bbox'] == pytest.approx([-74.09868, 4.434354, -73.10649, 5.425259], abs=tolerance)
+    assert result['crs'] == "4326"
```

Comment:
Can you add an assertion here that there is a user visible warning/log that you're defaulting to 4326 (if that is the case here, or elsewhere).